### DISCUSSION TITLE
2CH switch for shutter / does not show slider

### DISCUSSION THREAD

**Discussion by fraggle71** [arendst/Tasmota#18457] [Q&A]
Hi together,
i got a gosund sw9 2ch switch at one shutter and a shelly 2.5 at another shutter.
i tried to configure the 2ch switch for shutter mode as follows:

SetOption80 1
shuttermode 1
Interlock 1,2
Interlock ON

After reboot i do not get the shutter slider on the webpage and i cant use something like shutterposition or any calibration commands.

i am using the actual release of tasmota.

Why is that. Both shutters are the same and the shelly can be configured (shelly 2.5 original without tasmota). I see lots of videos where the slider and belonging commands ar used with a 2ch switch.

what am i missing ?

thanks for your help.

greetings matthias

**Comment by philfleck** [comment]
just a guess, maybe you did not set ```ShutterRelay1 1``` to let which realy is which as described here 
```
When SetOption80 1 is invoked you have to define Shutterrelay1 <value> to get started If possible to avoid any injury on unexpected movement all RELAYS should start in OFF mode when the device reboots: PowerOnState 0
```

https://tasmota.github.io/docs/Blinds-and-Shutters/#operation

**Reply by mjfuchs71** [reply]
Thanks. That's it.
